powerline/segments/common/bat.py

```diff
@@ -51,10 +50,16 @@ def _get_battery(pl):
 						pl.debug('Not using DBUS+UPower with {0}: not a power supply', devpath)
 						continue
 					pl.debug('Using DBUS+UPower with {0}', devpath)
-					return lambda pl: float(
```

Comment:
This function *must* return another function. I am not merging in a patch that removes caching.